app/src/main/res/layout/fragment_admin_generate_reports_page.xml

```diff
@@ -0,0 +1,91 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:background="@drawable/ggreen"
+    android:padding="16dp">
+
+    <TextView
+        android:id="@+id/title_text_view"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:text="Generate Reports"
+        android:textSize="24sp"
+        android:layout_centerHorizontal="true"
+        android:layout_marginBottom="16dp"
+        android:textStyle="bold" />
+
+    <Spinner
+        android:id="@+id/report_type_spinner"
+        android:layout_width="381dp"
+        android:layout_height="57dp"
+        android:layout_below="@id/title_text_view"
+        android:layout_marginTop="16dp"
+        android:prompt="@string/select_report_type_prompt" />
+
+    <LinearLayout
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_below="@id/report_type_spinner"
+        android:orientation="horizontal">
+
+        <TextView
+            android:id="@+id/start_date_label"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:text="Start Date:"
+            android:textSize="18sp"
+            android:layout_marginTop="16dp"
+            android:layout_marginEnd="8dp"/>
+
+        <DatePicker
+            android:id="@+id/start_date_picker"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_weight="1"
+            android:layout_marginTop="16dp"
+            android:layout_marginStart="8dp"
+            android:calendarViewShown="false"
+            android:datePickerMode="spinner"/>
+
+    </LinearLayout>
+
+    <LinearLayout
```

Comment:
This Linear Layout, overlaps the top linear layout, please fix

Reply:
done fix overlaps top linear layout